test/extensions/filters/http/wasm/test_data/test_grpc_call_cpp.cc

```diff
@@ -40,20 +40,20 @@ class GrpcCallRootContext : public RootContext {
   MyGrpcCallHandler* handler_ = nullptr;
 };
 
-class GrpcCallContext : public Context {
+class GrpcCallContextLegacy : public Context {
```

Comment:
Nit: It's in tests, so it doesn't really matter, but we should probably use "...Proto" and not "...Legacy".

Reply:
done